homework/calculate/calculate.hpp

```diff
@@ -2,6 +2,28 @@
 #include <string>
 
 std::string calculate(const std::string& command, int first, int second) {
-    // TODO: Implement your solution here and return proper value
-    return "";
+    std::string result{};
+    int calculation;
+
+    if(command.compare("add") == 0){
+        calculation = first + second;
+    }else if(command.compare("subtract") == 0){
+        calculation = first - second;
+    }else if(command.compare("multiply") == 0){
+        calculation = first * second;
+    }else if(command.compare("divide") == 0){
+        if(second != 0){
+            calculation = first / second;
+        }else{
+            result = "Division by 0";
+        }
+    }else{
+        result = "Invalid data";
+    }
```

Comment:
If you return variable here, you don't need to use `else if` or `else`

Add space between operations like '+' '-' '!=' '==' etc...
Add space between `else` `if` `for` `while` etc...